component/slider/slider.css

```diff
@@ -0,0 +1,26 @@
+.slider-enabled > input{
+    appearance: none;
+    background-color: var(--SECONDARY-COLOR);
+    border-radius: 10px;
+    width: 200px; 
```

Comment:
suggestion :- try avoiding px here and use rem,em,%

Reply:
Done.